fe/fe-core/src/main/java/com/starrocks/sql/common/MetaUtils.java

```diff
@@ -142,12 +142,54 @@ public static Table getTable(ConnectContext session, TableName tableName) {
         }
         Table table = session.getGlobalStateMgr().getMetadataMgr().getTable(tableName.getCatalog(),
                 tableName.getDb(), tableName.getTbl());
+
+        if (table == null) {
+            throw new SemanticException("Table %s is not found", tableName.toString());
+        }
+        return table;
+    }
+
+    // get table by tableName, unlike getTable, this interface is session aware,
+    // which means if there is a temporary table with the same name,
+    // use temporary table first, otherwise, treat it as a permanent table
+    public static Table getSessionAwareTable(ConnectContext session, TableName tableName) {
```

Comment:
```suggestion
    public static Table getTableSessionAware(ConnectContext session, TableName tableName) {
```

Reply:
session aware is an adjective, I don’t think it should be put at the end